crowbar_framework/app/controllers/installers_controller.rb

```diff
@@ -51,18 +51,40 @@ def start
       # the shell Process will be spawned in the background and therefore has
       # not a direct return value which we can use here
       if Crowbar::Installer.installing?
-        flash[:notice] = I18n.t(".installers.start.installation_ongoing")
+        msg = I18n.t(".installers.start.installation_ongoing")
+
+        respond_to do |format|
+          format.json {
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)